### PR TITLE
Fixes + Help + Feedback + App update alert

### PR DESCRIPTION
**Pull request by neetuy**
Added Hints and feedback to help section.
Added alert feature  to update the app(playstore) to the latest version.
Added Version Number at the bottom of the drawer.
Updated About Page.
Screenshot_2020-09-30-13-55-42-895_com vachango
Screenshot_2020-09-30-13-55-29-518_com vachango
Screenshot_2020-09-30-12-34-24-780_com vachango
Screenshot_2020-09-30-11-58-31-614_com vachango
Screenshot_2020-09-30-11-58-37-926_com vachango

### CODE REVIEW

File: app/screens/StudyHelp/Dictionary/index.js [hunk 1]
```diff
@@ -1,159 +1,77 @@
...
-    }
-    catch (error) {
-      this.setState({
-        wordDescription: [],
-        modalVisibleDictionary: false
-      })
+  async componentDidMount() {
+    const apiData = await APIFetch.getDictionaries()
+    if (apiData){
+      // console.log(" api data dictionary ",apiData,apiData.language,this.props.languageName.toLowerCase())
```
Comment (samuelanthrayose) [inline app/screens/StudyHelp/Dictionary/index.js:29]: Remove console.log

File: app/store/saga/apiFetch/fetchAllContent.js [hunk 2]
```diff
@@ -83,39 +83,40 @@ function* fetchAllContent() {
...
-          source: metaD && (metaD.hasOwnProperty("Source (Name & Year)") ? metaD["Source (Name & Year)"] : ''),
-          technologyPartner: metaD && (metaD.hasOwnProperty("Technology Partner") ? metaD["Technology Partner"] : ''),
-          versionName: metaD && (metaD.hasOwnProperty("Version Name (in Eng)") ? metaD["Version Name (in Eng)"] : ''),
-          versionNameGL: metaD && (metaD.hasOwnProperty("Version Name (in GL)") ? metaD["Version Name (in GL)"] : ''),
-        }]
-        versions.push({ sourceId: dictionariesLanguage[i].dictionaries[j].sourceId, versionName: dictionariesLanguage[i].dictionaries[j].name, versionCode: dictionariesLanguage[i].dictionaries[j].code, metaData: mData, downloaded: false })
-      }
-      dictionary.push({ languageName: language, languageCode: languageCode, versionModels: versions })
-    }
+    // for (var i = 0; i < dictionariesLanguage.length; i++) {
```
Comment (samuelanthrayose) [inline app/store/saga/apiFetch/fetchAllContent.js:86]: Are these commented lines needed?

**Review by samuelanthrayose — COMMENTED** [review]
@neetuy Have added 2 comments. Please check.

File: app/store/saga/apiFetch/fetchAllContent.js (hunk 2 above)
Comment (neetuy) [reply app/store/saga/apiFetch/fetchAllContent.js:86]: Ok, sure I'll remove it

File: app/screens/StudyHelp/Dictionary/index.js (hunk 1 above)
Comment (neetuy) [reply app/screens/StudyHelp/Dictionary/index.js:29]: ok, Sure I'll remove it

File: app/screens/StudyHelp/Dictionary/index.js [hunk 3]
```diff
@@ -1,159 +1,77 @@
...
+      for (var i = 0; i < apiData.length; i++) {
+      console.log(" api data dictionary ",apiData,apiData[i].language,this.props.languageName.toLowerCase())
```
Comment (samuelanthrayose) [inline app/screens/StudyHelp/Dictionary/index.js:30]: remove console.log

File: app/screens/StudyHelp/Dictionary/index.js [hunk 4]
```diff
@@ -1,159 +1,77 @@
...
-    this.errorMessage()
-    }
-    else{
-      return
-    }
-  }
-
-
   render() {
+    console.log("dictionaries ",this.state.dictionaries)
```
Comment (samuelanthrayose) [inline app/screens/StudyHelp/Dictionary/index.js:54]: remove console.log

**Pull request merged**